app/src/main/java/com/pixlee/pixleeandroidsdk/AppApplication.kt

```diff
@@ -11,5 +11,14 @@ class AppApplication: Application() {
         super.onCreate()
         // set credentials for the SDK
         PXLClient.initialize(BuildConfig.PIXLEE_API_KEY, BuildConfig.PIXLEE_SECRET_KEY)
+
+        // (Optional) if you use UI components and want
+        // to let PXLPhotoRecyclerViewInGrid, PXLPhotoRecyclerView and PXLPhotoProductView
+        // fire 'openedWidget', 'widgetVisible' and 'openedLightbox' analytics events, use this.
+        PXLClient.autoAnalyticsEnabled = true
```

Comment:
so it's set to true here, you don't pass it in somewhere?

Reply:
No. This place is where an app starts, same as Distillerys `CustomApplicationLoader.builder(context: ApplicationLoader.Context): GuiceApplicationBuilder`.